src/atomate2/settings.py

```diff
@@ -83,6 +83,12 @@ class Atomate2Settings(BaseSettings):
         "all the files are compressed. If 'atomate' only a selection of files related "
         "to the simulation will be compressed. If False no file is compressed.",
     )
+    VASP_INHERIT_INCAR: bool = Field(
+        False,
+        description="Whether to inherit INCAR settings from previous calculation. "
+        "This might be useful to port Custodian fixes to child jobs but can also be "
+        "dangerous e.g. when switching from GGA to meta-GGA or relax to static jobs.",
+    )
```

Comment:
As advocated for by @Andrew-S-Rosen during the atomate2 maintainer meeting, INCAR inheritance can not only be controlled at the `VaspInputGenerator` level but also using the new `VASP_INHERIT_INCAR` global flag.

Reply:
This has changed the default behaviour for all input sets. Please can you change this back to True and we can assess this as a separate issue. This discussion should probably happen on the pymatgen issues since the default pymatgen behaviour is to inherit incar settings and we will be merging the atomate2 inputs with those in pymatgen soon.

Reply:
Good point. `VASP_INHERIT_INCAR` now defaults to `True`.